ci/kokoro/windows/cmake-presubmit.cfg

```diff
@@ -13,5 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-build_file: "google-cloud-cpp-common/ci/kokoro/windows/bazel/build.bat"
-timeout_mins: 120
+env_vars {
+  key: "BUILD_CACHE"
+  value: "gs://cloud-cpp-kokoro-results/build-artifacts/common/vcpkg-installed-win-2019.zip"
```

Comment:
Nit: filename suggests it's using Windows 2019 image.

Reply:
Fixed, I think the new name makes it more obvious that it is related to the build more than anything else.